Source/DafnyStandardLibraries/Makefile

```diff
@@ -1,6 +1,6 @@
 
 # Invoking the CLI this way just to stay platform-independent
-DAFNY = dotnet run --project ../Dafny --no-build --
+DAFNY = dotnet run --project ../Dafny --no-build --roll-forward LatestMajor --
```

Comment:
I also picked `LatestMajor` before I noticed you'd already added some cases of `Major` instead. Are these unnecessary now that you've added the policy on all the .csproj files?